components/gorgias/gorgias.app.mjs

```diff
@@ -63,11 +93,11 @@ export default {
       };
     },
     _defaultConfig({
-      path, method = "get", params = {}, data = null,
+      path, method = "get", params = {}, data = undefined,
```

Comment:
Since non-passed data will be undefined, in my view is redundant to say `data=undefined`. The default value for a non-passed param is already `undefined`
```suggestion
      path, method = "get", params = {}, data,
```